stdlib/platform/java/binarysearchtree.wyv

```diff
@@ -0,0 +1,35 @@
+module def binarysearchtree(java : Java)
```

Comment:
put into an appropriate subdirectory

Reply:
Should I create and place this in a data-structures directory under java as well?

Reply:
Sure, but name it without the -.  Use camelCase if there are two words.